subprojects/detekt.yml

```diff
@@ -565,6 +565,8 @@ style:
     ExpressionBodySyntax:
         active: false
         includeLineWrapping: true
+    # https://detekt.github.io/detekt/style.html#forbiddencomment
+    # we use todos a lot, maybe there is a better way
```

Comment:
Maybe not in this PR but I'd love to use `STOPSHIP` or similar alternatives.

While coding I often use TODO in two different cases:
- Intentional tech debt. This is a valid usage, nothing to discuss.
- Marker for myself to implement something later but before PR. 
I don't want to miss it but not in all cases I can break the code for the fail-fast approach.
If I miss todo in the second case, it'll be harder to find.

Reply:
yes, i use this a lot too, but don't like it
i'd prefer separate tooling for such a debt like [codestream](https://www.codestream.com/), but it should be a process around this tool in team

another point is to comment about known issue that may help to debug it in future;

Reply:
> i'd prefer separate tooling for such a debt like codestream

Do you mean to use it for intentional tech debt and forbid TODOs at all?
I see no contradictions. We could choose something for the second case (STOPSHIP). We won't see such comments in PR, we won't merge such code. This comment wouldn't bother anyone at all who doesn't use it.

Reply:
yep, we don't have a process to keep todo's count close to zero, so it's only purpose now is to give a warning for next change about possible quirks
It could be just a regular comment

Reply:
Ok, I'll enable STOPSHIP'like stopper in an additional PR

Reply:
#956